src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/HmppsDomainEventsEmitterTest.kt

```diff
@@ -0,0 +1,65 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.services
+
+import com.amazonaws.services.sns.AmazonSNS
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.mockito.ArgumentMatchers.anyString
+import org.mockito.kotlin.any
+import org.mockito.kotlin.check
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.whenever
+import uk.gov.justice.digital.hmpps.prisonersearch.config.DiffProperties
+import uk.gov.justice.digital.hmpps.prisonersearch.services.diff.PropertyType
+import uk.gov.justice.hmpps.sqs.HmppsQueueService
+import uk.gov.justice.hmpps.sqs.HmppsTopic
+import java.time.Clock
+import java.time.Instant
+import java.time.ZoneId
+
+/*
+ * Most test scenarios are covered by the integration tests in HmppsDomainEventsEmitterIntTest
+ */
+class HmppsDomainEventsEmitterTest {
```

Comment:
Just need a couple of unit tests for scenarios not covered by the integration tests